README.md

```diff
@@ -147,7 +154,7 @@ To setup the test infrastructure, use `bundle exec rake acceptance:setup`. This
 
 Each setup step is its own task; `acceptance:setup`'s implementation consists of calling these tasks. Also, all setup tasks are idempotent. That means its safe to run them (and hence `acceptance:setup`) multiple times.
 
-**Note:** You can run the tests on a real ServiceNow instance. To do so, just invoke `bundle exec rake acceptance:setup_servicenow_instance[<fqdn>,<user>,<password>]`. This will update the `inventory.yaml` file with the actual ServiceNow instance credentials.
+**Note:** You can run the tests on a real ServiceNow instance. To do so, just invoke `bundle exec rake acceptance:setup_servicenow_instance[<fqdn>,<user>,<password>]`. This will update the `inventory.yaml` file with the actual ServiceNow instance credentials. You'll need to include a user/password and token to run all of the tests. The oauth tests will be skipped if no token is provided.
```

Comment:
Nit: `user/password and an optional oauth token to run all of the tests.`

Current wording makes it seem like the oauth token's required.